### ISSUE TITLE
Size errors with different nuclear and membrane channel selection for segmentation

### ISSUE THREAD

**Issue by ritamyhuang** [angelolab/ark-analysis#388]
**What are you trying to do?**
Run segmentation with "Segment_Image_Data.ipynb" on our own MIBI data, which is one multi-channel tiff file with 31 channels.

**What is the problem?**
Encounter errors when we select more one or more channels for the mems channels.
![image](https://user-images.githubusercontent.com/25540779/108816083-7c1fed00-75f0-11eb-8145-ae140183f868.png)
![image](https://user-images.githubusercontent.com/25540779/108818541-804e0980-75f4-11eb-9685-b33613b7242e.png)

**What have you tried so far?**
We have tried different lengths of nucs and mems, and they give different results as below:

nucs = ['dsDNA']
mems = ['Ecadherin', 'NaKATPase']
This yields ValueError: conflicting sizes for dimension 'channels': length 1 on the data but length 3 on coordinate 'channels'

nucs = ['dsDNA']
mems = ['Ecadherin', 'NaKATPase', 'CD45']
This yields ValueError: conflicting sizes for dimension 'channels': length 2 on the data but length 4 on coordinate 'channels'

nucs = None
mems = ['Ecadherin']
This yields ValueError: need at least one array to stack

nucs = ['dsDNA']
mems = None
This works, but we would like to be able to select mems.

**Additional context**
We did not encounter these errors with the older version of "Segment_Image_Data.ipynb" on our data, while the newest version gives this error. We have also tried the same data and script on a different computer, and the same error occurs.

**Comment by ngreenwald** [comment]
This error is saying that there is a different number of channels specified than the actual number of channels extracted from the MIBITIFF. My suspicion is that the channels might be spelled differently.
 
Can you try 
```
nucs = ['dsDNA']
mems = [ 'CD45']
```

and also 
```
nucs = ['dsDNA', 'Ecadherin']
mems = None
```

**Comment by ngreenwald** [comment]
Closed via #392. The script will not automatically detect if a channel has been spelled incorrectly, and throw an informative error. Feel free to reopen this issue if that wasn't the root cause.

**Comment by ritamyhuang** [comment]
Yes, misspelled channel names was the issue. Thanks for adding the detection feature!
